src/Module.Server/Modes/Dtv/CrpgDtvViscountUnderAttackMessage.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Text;
+using TaleWorlds.MountAndBlade;
+using TaleWorlds.MountAndBlade.Network.Messages;
+
+namespace Crpg.Module.Modes.Dtv;
+
+[DefineGameNetworkMessageTypeForMod(GameNetworkMessageSendType.FromServer)]
+internal sealed class CrpgDtvViscountUnderAttackMessage : GameNetworkMessage
+{
+    public Agent Attacker { get; set; } = default!;
```

Comment:
I think it's better to keep the property nullable because 
I think ReadAgentReferenceFromPacket could return null if the agent doesn't exist. It's very unlikely that this will ever happen but we never know.

Reply:
I've changed the property to nullable, let me know if you are happy with it and I will request some translations for the message to send in the event that the agent doesn't exist.